Jellyfin.Data/Entities/AncestorId.cs

```diff
@@ -0,0 +1,19 @@
+using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.ComponentModel.DataAnnotations.Schema;
+
+namespace Jellyfin.Data.Entities;
+
+#pragma warning disable CA1708 // Identifiers should differ by more than case
+#pragma warning disable CS1591 // Missing XML comment for publicly visible type or member
```

Comment:
New classes should have the properties documented instead of suppressing warnings